### ISSUE TITLE
Imperium - Dark Angels: "Special" Tactical Marines are missing their bolt pistols

### ISSUE THREAD

**Issue by BSDataAnon** [BSData/wh40k#6863]
**File:** Imperium - Dark Angels.catz

**BattleScribe version:** 2.03.12

**Platform:** Windows

**Dropbox:** No

**Description:** Dark Angel Tactical Marines, 

both Heavy Weapon and Special weapon troopers (additive tick boxes), are missing their bolt pistols

in the Codex when they take a 'weapon', heavy or special, they replace their bolter
if other sets have the same structure, I would assume that the same errors are there too
tl;dr the optional weapon marines, are supposed to have their bolt pistols too
